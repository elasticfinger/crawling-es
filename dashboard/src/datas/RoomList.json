{
    "rooms": [{
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 2,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id":3,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 1,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }, {
        "id": 100,
        "uid": "bjbjbjbjbj",
        "date": "20190101"
    }],
    "lastPage": 1000
}