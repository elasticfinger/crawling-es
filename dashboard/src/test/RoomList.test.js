import React from 'react'
import { render, fireEvent } from '@testing-library/react'
import RoomList from '../pages/rooms'
import axios from 'axios'
import { act } from 'react-dom/test-utils';
import datas from '../datas/RoomList.json'
import {BrowserRouter} from "react-router-dom"
import ReactDOM from 'react-dom';
const URL = '/main/rooms'

jest.mock('axios')
jest.mock('react-router-dom', () => {
    const originalModule = jest.requireActual('react-router-dom');
    return {
        __esModule: true,
        ...originalModule,
        useParams: () => ({
            page: 1,
            search: undefined
        }),
        useHistory: () => ({
            push: jest.fn()
        }),
    };
})


test('Room List Snapshot', async () => {
    let container;
    axios.get = jest.fn().mockResolvedValue({
        data: datas
    })
    await act(async () => {
        container = render(<BrowserRouter><RoomList /></BrowserRouter>).container
    })
    expect(container).toMatchSnapshot();
})

test('search input', async () => {
    let container;

    await act(async () => {
        container = render(<BrowserRouter><RoomList /></BrowserRouter>).container
    })
    const searchInput = container.querySelector('input')
    const input ={ target: { value: 'test' } }
    await act(async () => {
        fireEvent.change(searchInput,input)
    })
    
    expect(searchInput).toHaveValue('test')

})
