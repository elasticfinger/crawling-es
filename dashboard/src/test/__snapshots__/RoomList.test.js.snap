// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Room List Snapshot 1`] = `
<div>
  <div
    class="sc-AxgMl jWcWDi"
  >
    <div
      class="sc-fzqNJr ivGJQY"
    >
      <div
        class="sc-AxhCb gnKBgn"
        height="150px"
      >
        <div
          class="sc-fzoyAV gHA-DHF"
        >
          <div
            class="sc-AxhCb jVMcxD"
          >
            <h2>
              Dashboard
            </h2>
          </div>
        </div>
      </div>
      <div
        class="sc-AxhCb jVMcxD"
      >
        <ul
          class="sc-fzoLag gIvpES"
        >
          <a
            href="/main/home"
          >
            <li
              class="sc-fzoXzr fJUFzM none"
            >
              <i
                class="fas fa-home"
              />
              홈
            </li>
          </a>
          <a
            href="/main/rooms/1"
          >
            <li
              class="sc-fzpjYC dbVUwD nav_ul-select"
            >
              <i
                class="fas fa-list"
              />
              채팅목록조회
            </li>
          </a>
        </ul>
      </div>
    </div>
    <div
      class="sc-AxheI cOUrdz"
      id="scroll-wrapper"
    >
      <header
        class="sc-AxjAm kiisUO"
      >
        <div
          class="sc-AxirZ fGikRl"
        >
          <div
            class="sc-AxiKw uuelv"
          >
            <div />
          </div>
          <div
            class="sc-AxiKw uuelv"
          >
            <div>
              <button>
                로그아웃
              </button>
            </div>
          </div>
        </div>
      </header>
      <article
        class="sc-AxhUy gHhRwB"
      >
        <div
          class="sc-fzplWN jsulIf"
        >
          <div
            class="sc-Axmtr gyjsBZ"
          >
            <div
              class="searchWrapper"
            >
              <input
                class="sc-fznZeY eDdIMr"
                value=""
              />
              <button
                class="sc-fznKkj hXewAi"
              >
                찾기
              </button>
            </div>
          </div>
          <div
            class="sc-fzpans dnlZUu"
          >
            <div
              class="sc-fzozJi epZuwd"
            >
              <div
                class="sc-AxmLO sc-fzoLsD iaELoE"
              >
                <div>
                  글번호
                </div>
                <div>
                  BJ명
                </div>
                <div>
                  방송시작시간
                </div>
              </div>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/2"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    2
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/3"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    3
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/1"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    1
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
              <a
                href="/main/room/100"
              >
                <div
                  class="sc-fznyAO kqrifC"
                >
                  <div>
                    100
                  </div>
                  <div>
                    bjbjbjbjbj
                  </div>
                  <div>
                    20190101
                  </div>
                </div>
              </a>
            </div>
            <div
              class="sc-fzokOt kcJeBe"
            >
              <div
                class="sc-fzqBZW cmiMx currentPage"
              >
                <a
                  href="/main/rooms/1"
                >
                  1
                </a>
              </div>
              <div
                class="sc-fzqBZW cmiMx"
              >
                <a
                  href="/main/rooms/2"
                >
                  2
                </a>
              </div>
              <div
                class="sc-fzqBZW cmiMx"
              >
                <a
                  href="/main/rooms/3"
                >
                  3
                </a>
              </div>
              <div
                class="sc-fzqBZW cmiMx"
              >
                <a
                  href="/main/rooms/4"
                >
                  4
                </a>
              </div>
              <div
                class="sc-fzqBZW cmiMx"
              >
                <a
                  href="/main/rooms/5"
                >
                  5
                </a>
              </div>
              <div
                class="sc-fzqBZW cmiMx"
              >
                <a
                  href="/main/rooms/6"
                >
                  6
                </a>
              </div>
              <div
                class="sc-fzqBZW cmiMx"
              >
                <a
                  href="/main/rooms/7"
                >
                  7
                </a>
              </div>
              <div
                class="sc-fzqBZW cmiMx"
              >
                <a
                  href="/main/rooms/8"
                >
                  8
                </a>
              </div>
              <div
                class="sc-fzqBZW cmiMx"
              >
                <a
                  href="/main/rooms/9"
                >
                  9
                </a>
              </div>
              <div
                class="sc-fzqBZW cmiMx"
              >
                <a
                  href="/main/rooms/10"
                >
                  10
                </a>
              </div>
              <div
                class="sc-fzqBZW cmiMx"
              >
                <a
                  href="/main/rooms/11"
                >
                  &gt;
                </a>
              </div>
            </div>
          </div>
        </div>
      </article>
    </div>
  </div>
</div>
`;
